docs-app/tests/kolay/components/component-signature-test.gts

```diff
@@ -147,6 +147,7 @@ module('<ComponentSignature>', function (hooks) {
       </template>
     );
 
+    await waitUntil(() => this.element.textContent.includes('Element:hehe'));
```

Comment:
thanks for fixing my tests :see_no_evil: 

I'm not sure where I missed a waiter -- I'll have to look at that later